index.js

```diff
@@ -144,6 +163,10 @@ function makeError(result, options) {
 	error.cmd = joinedCommand;
 	error.timedOut = Boolean(timedOut);
 
+	if ('all' in result) {
```

Comment:
Why is this guarded? It should not be treated differently than `stdout` and `stderr`, unless I'm missing something?

Reply:
I did this because the `makeError` function is used by `execa.sync()` as well, which does not have the `all` stream in `result.all`. This guard was meant to prevent adding `all` property when it's not available on the `result` object at all.
I used the `in`, which is stricter, so that it guards from using `in` and `Object#hasOwnProperty` on the error object to, for example, check if the error came from the async or sync execa. But I don't have any other use cases in which it can be useful.
What I meant in the code was to highlight that it's not part of the original result and that it's more of an addition. This guard may be too much.

Reply:
This makes sense to me.